tests/rados/pool_tests.py

```diff
@@ -1095,20 +1131,18 @@ def run(ceph_cluster, **kw):
                 time.sleep(10)
 
                 # verify IOs are resumed after sometime post setting min_size to 1
-                pool_obj.do_rados_put(
-                    client=client_node, pool=pool_name, nobj=10, timeout=10
+                if not pool_obj.do_rados_get(pool=pool_name, read_count=20):
+                    log.error(f"Unable perform read operations on the pool {pool_name}")
+                    raise Exception("Reads failed on the cluster Error")
+
+                if pool_obj.do_rados_put(client=client_node, pool=pool_name, nobj=5):
+                    log.error("UnAble to perform writes at min_size. Error")
+                    raise Exception("Writes failed on the cluster Error")
+
+                log.info(
+                    "Unable to perform read & write operations on the cluster as expected"
+                    " Proceeding to bring down another OSD on the PG"
                 )
-                pool_stat_post = rados_obj.get_cephdf_stats(pool_name=pool_name)
-                log.debug(pool_stat_post)
-                if pool_stat_post["stats"]["objects"] <= total_objs:
-                    log.error(
-                        "Write ops should have been possible, number of objects in the pool have NOT changed"
-                    )
-                    raise Exception(
-                        f"Pool {pool_name} has {pool_stat_post['stats']['objects']} objs"
-                    )
-                else:
-                    log.info("RADOS PUT op is successful as expected!! ")
```

Comment:
Any reason to remove this validation?

Reply:
redundant checks. Can be added back if needed.

Reply:
Added the checks back.